src/ros2/usb_cam_node.cpp

```diff
@@ -171,7 +171,7 @@ void UsbCamNode::init()
 
   // if pixel format is equal to 'mjpeg', i.e. raw mjpeg stream, initialize compressed image message
   // and publisher
-  if (m_parameters.pixel_format_name == "mjpeg") {
+  if (m_parameters.pixel_format_name == "raw_mjpeg") {
```

Comment:
@hilookas We should support both mjpeg and raw_mjpeg for compressed image streams I think, no?

```suggestion
  auto pixel_fmt = m_parameters.pixel_format_name;
  if (pixel_fmt == "mjpeg" || pixel_fmt == "raw_mjpeg" ) {
```